gulpfile.js

```diff
@@ -128,10 +131,36 @@ gulp.task('lint', function () {
         .pipe(gulpif(isFixed, gulp.dest('.')));
 });
 
-gulp.task('gh-deploy', ['switch-to-src','default'], () => {
+gulp.task('gh-deploy', (cb) => {
+    if (!argv.remoteUrl) {
+        return Promise.reject('--remoteUrl="..." flag is required');
+    }
+    return sequence('gh-deploy-confirm', ['switch-to-src','default'], 'gh-deploy-helper')(cb);
+});
+gulp.task('gh-deploy-confirm', () => {
+    if ('https://github.com/MergeMyPullRequest/order-splitter.git' === argv.remoteUrl) {
+        return inquirer.prompt([
+            {
+                type: 'confirm',
+                name: 'shouldDeploy',
+                message: 'Are you sure you want to deploy to "mergemypullrequest.github.io/order-splitter"?'
+            }
+        ]).then(({shouldDeploy}) => {
+            if (!shouldDeploy) {
+                throw new Error('cancelled by user');
+            } else {
+                remoteUrl = argv.remoteUrl; // add to global scope
+            }
+        });
+    } else {
+        remoteUrl = argv.remoteUrl; // add to global scope
+        return Promise.resolve();
+    }
+});
+gulp.task('gh-deploy-helper', () => {
     return gulp.src(deployDir+'/**')
         .pipe(ghPages({
```

Comment:
If you `cd` into `/.publish` and run `git remote -vv`, you'll notice that there's only 1 remote: "origin". Not sure why they bothered making this an argument.